beacon-chain/node/node.go

```diff
@@ -662,3 +669,79 @@ func (b *BeaconNode) registerArchiverService() error {
 	})
 	return b.services.RegisterService(svc)
 }
+
+func loadChainConfigFile(chainConfigFileName string) {
+	yamlFile, err := ioutil.ReadFile(chainConfigFileName)
+	if err != nil {
+		log.WithError(err).Error("Failed to read chain config file.")
+	}
+	// convert 0x hex inputs to fixed bytes arrays
+	lines := strings.Split(string(yamlFile), "\n")
```

Comment:
can you refactor this whole chunk into a separate method and unit test it

Reply:
done, thanks